mission-control-client/src/components/auth/Login.js

```diff
@@ -123,6 +123,7 @@ const FormikLogin = withFormik({
       .catch(err => {
         setSubmitting(false)
         setStatus(err.response.data.message)
+        // setStatus('Error')
```

Comment:
We'll have to discard these changes because they were for debugging.

Reply:
done